src/main/java/life/qbic/presenter/charts/WorkflowPresenter.java

```diff
@@ -40,14 +57,14 @@ void addChartData() {
         Object[] objectArray = chartConfig.getData().keySet().toArray(new Object[chartConfig.getData().keySet().size()]);
         String[] keySet = Arrays.asList(objectArray).toArray(new String[objectArray.length]);
 
+        Color[] innerColors = Arrays.copyOf(Helper.colors, chartConfig.getSettings().getxCategories().size());
```

Comment:
If I were looking for coloring I don't think that I would look for that in a class called Helper. We have a "util" package, since the Helper class is the only class there I guess that we could create classes which contain solely one/two.. helper methods and name them accordingly. 
Since we're working on this currently we know what the Helper class does and where we can get the coloring from, but others likely won't.